content/en/account_management/billing/custom_metrics.md

```diff
@@ -7,64 +7,200 @@ aliases:
 
 ## Overview
 
-In general, [custom metrics][1] refers to any metrics that are not part of the regular integration suite, for example using custom checks or API-level metrics in your application stack.
+If a metric is not submitted from one of the [350+ Datadog integrations][1] it's considered a [custom metric][2]<sup>[(1)](#standard-integrations)</sup>.
```

Comment:
Can we add something about what a custom metric _is_, rather than strictly what it is not, or a link to a neat blog?  `Custom metrics measure often telemetry about an application`, `Examples of custom metrics include critical business telemetry like time on page, or other operational data about the behavior of a customer application` or something?